coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -4795,3 +4795,27 @@ def remainder(context, node):
     context.add(scaled_div)
     remainder_node = mb.sub(x=dividend, y=scaled_div, name=node.name)
     context.add(remainder_node)
+
+
+@register_torch_op
+def hann_window(context, node):
+    inputs = _get_inputs(context, node, expected=5)
+    # TODO: Support unknown window length
+    # TODO: Support unknown periodic
+    size = (inputs[0].val,)
+    if inputs[0].val == 1:
+        one = mb.fill(shape=size, value=1.0, name=node.name)
+        context.add(one)
+        return
+
+    ones = mb.fill(shape=size, value=1.0)
+    cum = mb.cumsum(x=ones, axis=0)
+    seq = mb.sub(x=cum, y=ones)
+    pi = mb.fill(shape=size, value=_math.pi)
+    window_length_float = mb.cast(x=inputs[0], dtype="fp32")
```

Comment:
The denominator is `N - 1`.
https://pytorch.org/docs/stable/generated/torch.hann_window.html
Should it be `x=inputs[0].val - 1`?

Reply:
Ok. Sounds good.